apidoc/Titanium/Platform/Platform.yml

```diff
@@ -208,6 +208,8 @@ properties:
     summary: |
         Name of the platform. Returns `android` for the Android platform, `iPhone OS` for the iOS
         platform (iPhone, iPad, or iPod Touch), `windows` for the Windows platform, and `mobileweb` for the Mobile Web platform.
+        
+        Since 5.5.0.GA return 'iOS' for for the iOS platform (iPhone, iPad, or iPod Touch).
```

Comment:
I would rephrase it to:

> Since 5.5.0.GA and Xcode 8, this property returns `iOS` for for the iOS platform (iPhone, iPad, or iPod Touch).

Are you fine with that? And good catch btw!